specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -6,9 +6,19 @@ This is the AutoRest configuration file the Cognitive Services Language SDK.
 
 ## Releases
 
-The current preview release is 2022-02-01-preview
+The current preview release of Conversational Language Understanding and Text Analytics is 2022-03-01-preview.
```

Comment:
This will likely have to be updated every time, so I recommend just keeping the text besides the date/api-version general.